mdata/store_cassandra.go

```diff
@@ -362,7 +366,12 @@ func (o asc) Less(i, j int) bool { return o[i].sortKey < o[j].sortKey }
 
 func (c *CassandraStore) processReadQueue() {
 	for crr := range c.readQueue {
-		cassGetWaitDuration.Value(time.Since(crr.timestamp))
+		waitDuration := time.Since(crr.timestamp)
+		cassGetWaitDuration.Value(waitDuration)
+		if waitDuration > c.omitReadTimeout {
```

Comment:
so to fix that problem i'll also need to do what you mentioned earlier with the non-blocking adding to the queue. i thought that's out of the scope of this PR, but i can add it too